extensions/wikia/IndexingPipeline/PipelineEventProducer.class.php

```diff
@@ -1,69 +1,173 @@
 <?php
 
 class PipelineEventProducer {
+	const WG_CONTENT_NAMESPACES_KEY = 'wgContentNamespaces';
```

Comment:
If you decide to go with https://github.com/Wikia/app/pull/8862/files?diff=split#diff-51080aaef54b01dd7a86c59c827a2761R164, this will not be necessary